google/cloud/bigtable/table.cc

```diff
@@ -48,6 +48,7 @@ static_assert(std::is_copy_assignable<bigtable::Table>::value,
 
 Status Table::Apply(SingleRowMutation mut) {
   if (connection_) {
+    google::cloud::internal::OptionsSpan span(options_);
```

Comment:
Could/should these be outside the `if (connection_)` conditional?

Reply:
The spans could be outside the if conditionals, but there is no point. The code outside of the if conditionals does not care about `CurrentOptions()`. I'd rather have them inside the if blocks.

Reply:
> The code outside of the if conditionals does not care about `CurrentOptions()`.

OK, although I'll assert that it is difficult to know that, and it may be better to be safe than sorry.